crates/bevy_core_pipeline/src/oit/resolve/mod.rs

```diff
@@ -61,6 +64,17 @@ impl Plugin for OitResolvePlugin {
             return;
         }
 
+        if render_app
+            .world()
+            .resource::<RenderDevice>()
+            .limits()
+            .max_storage_buffers_per_shader_stage
+            < OIT_REQUIRED_STORAGE_BUFFERS
+        {
+            warn!("OrderIndependentTransparencyPlugin not loaded. RenderDevice lacks support: max_storage_buffers_per_shader_stage < OIT_REQUIRED_STORAGE_BUFFERS.");
+            return;
```

Comment:
It might be nice to return after all compatibility checks are printed, but this is good enough as is